doc/classes/AudioStreamWAV.xml

```diff
@@ -24,18 +24,19 @@
 		<member name="data" type="PackedByteArray" setter="set_data" getter="get_data" default="PackedByteArray()">
 			Contains the audio data in bytes.
 			[b]Note:[/b] This property expects signed PCM8 data. To convert unsigned PCM8 to signed PCM8, subtract 128 from each byte.
+			If the audio format is set to Quite OK Audio, it expects data from a QOA file.
```

Comment:
This is another note, technically. Although I'm not sure how necessary this is, are the other modes any different? It "reads" like it's obvious.

```suggestion
			[b]Note:[/b] If [member format] is set to [constant FORMAT_QOA], it expects data from a QOA file.
```

Reply:
I'd like to add something about this entire description.

```
[b]Note:[/b] This property expects signed PCM8 data. To convert unsigned PCM8 to signed PCM8, subtract 128 from each byte.
```
Normally, 8-bit PCM WAVs are unsigned. This note exists because of two things:
- When a WAV file is imported in 8-bit format, the importer internally converts it from unsigned to signed (subtracting 128 from each byte) so it's easier to convert to 16-bit for playback (all needed is a bit shift).
- When you create an AudioStreamWAV from code, 8-bit PCM format is the default, so the note actually applies just to that. 

IMA ADPCM just requires sample data in that format, while QOA requires an entire QOA file data.

Wouldn't it be better to write all this info at once?

Reply:
I suppose it would, yes. As you wrote it now it feels less obvious. If I may recommend, you could embed a link to QOA's website which should teach how to decode the bytes if needed.